### PR TITLE
Add docker compose

### LINKED ISSUE #206: 建议增加一个 docker 方式的快速开始
## In what area(s)?

> /area docs

## Describe the feature

可以弄一个 docker-compose 脚本，让用户可以快速的把 sofa-registry 快速运行起来。

### PR DESCRIPTION
**Pull request by stulzq**
### Motivation:

Integrating mysql, you can quickly start sofa registry through docker compose without installing the database locally.

### Modification:

Add docker compose startup.

### Result:

Fixes #206

### CODE REVIEW

File: docker/compose/sofa-registry-in-docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+version: '3.3'
+services:
+  sofa-regisstry:
```
Comment (dzdx) [inline docker/compose/sofa-registry-in-docker-compose.yml:3]: `sofa-registry` 是不是拼错了

File: docker/compose/sofa-registry-in-docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      JDBC_URL: jdbc:mysql://sofa-mysql:3306/registrymetadb?useSSL=false&useUnicode=true&characterEncoding=utf8
+      JDBC_USERNAME: root
+      JDBC_PASSWORD: root
+      REGISTRY_APP_NAME: integration
+    networks:
+      - sofa-regisrty-ns
+  sofa-mysql:
+    image: mysql:5.7
+    command: --init-file /docker-entrypoint-initdb.d/init.sql --default-authentication-plugin=mysql_native_password --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
+    container_name: sofa-regisstry-mysql
```
Comment (dzdx) [inline docker/compose/sofa-registry-in-docker-compose.yml:25]: 这边

File: docker/compose/sofa-registry-in-docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    image: sofaregistry/sofaregistry:6.1.4
+    container_name: sofa-registry-integration
+    depends_on:
+      - sofa-mysql
+    links:
+      - sofa-mysql
+    restart: always
+    ports:
+      - 9615:9615
+      - 9622:9622
```
Comment (dzdx) [inline docker/compose/sofa-registry-in-docker-compose.yml:13]: 暴露一下9600端口，client连接用的bolt的端口

File: docker/compose/sofa-registry-in-docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      JDBC_URL: jdbc:mysql://sofa-mysql:3306/registrymetadb?useSSL=false&useUnicode=true&characterEncoding=utf8
+      JDBC_USERNAME: root
+      JDBC_PASSWORD: root
+      REGISTRY_APP_NAME: integration
+    networks:
+      - sofa-regisrty-ns
+  sofa-mysql:
+    image: mysql:5.7
+    command: --init-file /docker-entrypoint-initdb.d/init.sql --default-authentication-plugin=mysql_native_password --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
+    container_name: sofa-regisrty-mysql
```
Comment (bjxiaojian) [inline docker/compose/sofa-registry-in-docker-compose.yml:26]: regisrty -> registry

**Review by dzdx — APPROVED** [review]
LGTM

**Pull request merged**